src/OrchardCore.Modules/OrchardCore.OpenId/Drivers/OpenIdValidationSettingsDisplayDriver.cs

```diff
@@ -72,7 +72,10 @@ public override async Task<IDisplayResult> UpdateAsync(OpenIdValidationSettings
 
         if (string.IsNullOrWhiteSpace(model.Tenant))
```

Comment:
While we're at it, let's use `string.IsNullOrEmpty()` instead of `string.IsNullOrWhiteSpace()` lines 73 and 75 😃

Reply:
Done!